jsx/PaginationLinks.js

```diff
@@ -1,29 +1,30 @@
 /* exported RPaginationLinks */
 
-let PaginationLinks = React.createClass({
-  mixins: [React.addons.PureRenderMixin],
```

Comment:
what is this guy and did you get rid of it?

Reply:
PureRenderMixin? It used to be a mixin for react components where render is a pure function of state and props which made it more efficient, but I think it was deprecated a couple React releases ago.

Reply:
@zaliqarosli It's no longer needed with what @driusan said.